Source/DefaultValue.cs

```diff
@@ -40,18 +40,20 @@
 
 namespace Moq
 {
+    using System;
+
 	/// <summary>
 	/// Determines the way default values are generated 
 	/// calculated for loose mocks.
 	/// </summary>
-	public enum DefaultValue
+	public static class DefaultValue
 	{
 		/// <summary>
 		/// Default behavior, which generates empty values for 
 		/// value types (i.e. default(int)), empty array and 
 		/// enumerables, and nulls for all other reference types.
 		/// </summary>
-		Empty,
+		public static readonly Func<IDefaultValueProvider> Empty = () => new EmptyDefaultValueProvider();
```

Comment:
This would look better in intellisense as a static property, initialized from the static ctor